src/GitVersion.LibGit2Sharp/Git/GitRepository.cs

```diff
@@ -53,6 +55,30 @@ public void DiscoverRepository(string? gitDirectory)
         });
     }
 
+    public IEnumerable<string>? FindPatchPaths(ICommit commit, string? tagPrefix)
+    {
+        Patch? patch = null;
+        var innerCommit = this.RepositoryInstance.Commits.First(c => c.Sha == commit.Sha);
+        var match = new Regex($"^({tagPrefix ?? ""}).*$", RegexOptions.Compiled);
```

Comment:
please do not create Regex in line, consider moving it to https://github.com/GitTools/GitVersion/blob/main/src/GitVersion.Core/Core/RegexPatterns.cs